internal/commands/user/list/flags.go

```diff
@@ -0,0 +1,65 @@
+package list
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	flagState      = "state"
+	flagStateShort = "s"
+	flagStateUsage = `select the state of users to list, available options: ["enabled", "disabled"]`
+
+	flagStatus      = "status-pending"
+	flagStatusShort = "p"
+	flagStatusUsage = `select the state of users to list, available options: ["enabled", "disabled"]`
+
+	flagProviderTypes      = "provider"
```

Comment:
similarly here, can we rename the vars to match the flag value, so it's `flagProvider`, `flagProviderShort`, ...